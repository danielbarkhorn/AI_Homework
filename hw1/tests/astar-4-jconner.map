50 60 56 54 61 61 58 58 59 59
59 50 49 43 44 63 67 70 70 55
66 54 42 46 36 51 61 65 68 57
72 62 57 56 38 53 60 66 65 62
62 66 70 61 50 47 63 58 56 56
53 58 54 52 58 46 50 63 63 52
47 44 39 38 40 53 52 67 65 65
42 52 41 45 46 41 51 69 76 64
41 56 46 45 52 45 57 54 57 68
