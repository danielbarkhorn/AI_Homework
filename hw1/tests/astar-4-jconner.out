Solution steps: ['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'E', 'N', 'N', 'E']
Solution cost: 283
Number of states considered: 74

Frontier:
Pos=(7, 3) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'S', 'E', 'S'] Cost=278
Pos=(7, 8) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'E', 'N', 'N', 'W'] Cost=284
Pos=(6, 7) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'N'] Cost=275
Pos=(4, 8) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N', 'E', 'N', 'E', 'E'] Cost=280
Pos=(7, 7) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'E', 'N', 'W'] Cost=271

Closed List:
Pos=(0, 0) Moves=[] Cost=0
Pos=(0, 1) Moves=['N'] Cost=2
Pos=(0, 2) Moves=['N', 'N'] Cost=28
Pos=(1, 2) Moves=['N', 'N', 'E'] Cost=32
Pos=(2, 2) Moves=['N', 'N', 'E', 'E'] Cost=38
Pos=(3, 2) Moves=['N', 'N', 'E', 'E', 'E'] Cost=40
Pos=(2, 1) Moves=['N', 'N', 'E', 'E', 'S'] Cost=43
Pos=(4, 2) Moves=['N', 'N', 'E', 'E', 'E', 'E'] Cost=45
Pos=(0, 3) Moves=['N', 'N', 'N'] Cost=65
Pos=(3, 1) Moves=['N', 'N', 'E', 'E', 'S', 'E'] Cost=60
Pos=(4, 1) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'E'] Cost=62
Pos=(3, 0) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'S'] Cost=61
Pos=(2, 0) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'S', 'W'] Cost=63
Pos=(5, 1) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'E', 'E'] Cost=68
Pos=(1, 3) Moves=['N', 'N', 'N', 'E'] Cost=91
Pos=(5, 0) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'E', 'E', 'S'] Cost=85
Pos=(2, 3) Moves=['N', 'N', 'N', 'E', 'E'] Cost=96
Pos=(3, 3) Moves=['N', 'N', 'N', 'E', 'E', 'E'] Cost=99
Pos=(1, 1) Moves=['N', 'N', 'E', 'S'] Cost=97
Pos=(4, 0) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'E', 'S'] Cost=99
Pos=(1, 0) Moves=['N', 'N', 'E', 'S', 'S'] Cost=114
Pos=(4, 3) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E'] Cost=136
Pos=(0, 4) Moves=['N', 'N', 'N', 'N'] Cost=147
Pos=(4, 4) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N'] Cost=145
Pos=(5, 4) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E'] Cost=149
Pos=(5, 3) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'E'] Cost=149
Pos=(1, 4) Moves=['N', 'N', 'N', 'E', 'N'] Cost=156
Pos=(1, 5) Moves=['N', 'N', 'N', 'E', 'N', 'N'] Cost=161
Pos=(2, 5) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'E'] Cost=167
Pos=(3, 5) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'E', 'E'] Cost=169
Pos=(6, 3) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E'] Cost=166
Pos=(1, 6) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N'] Cost=170
Pos=(6, 1) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'E', 'E', 'E'] Cost=169
Pos=(4, 5) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'N'] Cost=158
Pos=(6, 2) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'S'] Cost=171
Pos=(5, 2) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'E', 'E', 'S', 'W'] Cost=173
Pos=(4, 6) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'N', 'N'] Cost=161
Pos=(3, 4) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'N'] Cost=181
Pos=(1, 7) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N'] Cost=175
Pos=(2, 4) Moves=['N', 'N', 'N', 'E', 'N', 'E'] Cost=173
Pos=(2, 7) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N', 'E'] Cost=177
Pos=(5, 5) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N'] Cost=186
Pos=(3, 6) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'E', 'E', 'N'] Cost=180
Pos=(5, 6) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'N'] Cost=189
Pos=(3, 7) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N', 'E', 'E'] Cost=184
Pos=(4, 7) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N', 'E', 'E', 'E'] Cost=186
Pos=(2, 6) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'E', 'N'] Cost=183
Pos=(6, 0) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'E', 'E', 'E', 'S'] Cost=206
Pos=(7, 0) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'E', 'E', 'E', 'S', 'E'] Cost=210
Pos=(8, 0) Moves=['N', 'N', 'E', 'E', 'S', 'E', 'E', 'E', 'E', 'S', 'E', 'E'] Cost=220
Pos=(2, 8) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N', 'E', 'N'] Cost=227
Pos=(3, 8) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N', 'E', 'N', 'E'] Cost=230
Pos=(6, 5) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E'] Cost=236
Pos=(6, 6) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N'] Cost=238
Pos=(1, 8) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N', 'E', 'N', 'W'] Cost=244
Pos=(6, 4) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'S'] Cost=246
Pos=(7, 4) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'S', 'E'] Cost=252
Pos=(8, 4) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'S', 'E', 'E'] Cost=255
Pos=(9, 4) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'S', 'E', 'E', 'E'] Cost=256
Pos=(7, 6) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E'] Cost=255
Pos=(0, 7) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N', 'W'] Cost=257
Pos=(7, 5) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'S'] Cost=257
Pos=(8, 5) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'S', 'E'] Cost=259
Pos=(9, 5) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'S', 'E', 'E'] Cost=263
Pos=(0, 5) Moves=['N', 'N', 'N', 'N', 'N'] Cost=248
Pos=(0, 8) Moves=['N', 'N', 'N', 'E', 'N', 'N', 'N', 'N', 'E', 'N', 'W', 'W'] Cost=255
Pos=(9, 3) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'S', 'E', 'E', 'E', 'S'] Cost=261
Pos=(9, 6) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'S', 'E', 'E', 'N'] Cost=269
Pos=(0, 6) Moves=['N', 'N', 'N', 'N', 'N', 'N'] Cost=255
Pos=(8, 6) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'E'] Cost=265
Pos=(9, 7) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'S', 'E', 'E', 'N', 'N'] Cost=272
Pos=(8, 7) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'E', 'N'] Cost=270
Pos=(8, 8) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'E', 'N', 'N'] Cost=282
Pos=(9, 8) Moves=['N', 'N', 'N', 'E', 'E', 'E', 'E', 'N', 'E', 'N', 'E', 'N', 'E', 'E', 'N', 'N', 'E'] Cost=283
